File: parallelSortJava/src/main/ArrayGenerator.java
package main;

import java.util.Random;

public class ArrayGenerator {

    public static int[] gen(int length, int low, int high) {
        Random rand = new Random();
        int[] output = new int[length];

        for (int i = 0; i < length; i++) {
            output[i] = rand.nextInt(high - low) + low;
        }

        return output;
    }

}

File: parallelSortGo/src/main/ArrayGenerator.go
package main

import (
	"math/rand"
	"time"
)

func ArrayGen(length int, low int, high int) []int {
	output := make([]int, length)
	source := rand.NewSource(time.Now().UnixNano())
	sRand := rand.New(source)

	for i := 0; i < length; i++ {
		output[i] = sRand.Intn(high-low) + low
	}

	return output
}
